src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  justify-content: center;
+  gap: 48px;
+
+  @media (min-width: 1190px) {
```

Comment:
are you sure it should be 1190? check the task again

Reply:
There should be four columns starting at 1024px
There should be three columns starting at 768px
There should be two columns starting at 488px